src/components/scrollManager.js

```diff
@@ -223,7 +223,7 @@ import layoutManager from 'layoutManager';
             let parent = element.parentElement;
 
             while (parent) {
-                // Skip 'emby-scroller' because it scrolls by itself
+                // Skip '../../elements/emby-scroller/emby-scroller' because it scrolls by itself
                 if (!parent.classList.contains('emby-scroller') &&
```

Comment:
I think this is another find/replace error lol

Reply:
reverted comment